Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
     KERNEL_ARTIFACTS_PATH = 'kernel-artifacts'
     UBUNTU_VERSION = '16'
     BUILD_PATH = '/mnt/tmp/kernel-build-folder'
-    KERNEL_CONFIG = './Microsoft/config-azure'
```

Comment:
commit message:
Change default kernel config to the Azure one

Reply:
If the kernel config path parameter is set, one of the following steps will be executed:
Step 1: If the config exists relative to the kernel sources folder, it will be copied to the kernel sources `.config` file. If config does not exist relative to the kernel sources, go to Step 2. 
Step 2.  If the config exists relative to the lis-pipeline/scripts/package_building/deps/kernel_config folder, it will be copied to the kernel sources `.config` file. If config does not exist relative to the lis-pipeline/scripts/package_building/deps/kernel_config folder, go to Step 3.
Step 3. Generate a `.config` file by executing `make olddefconfig`